apps/molgenis-components/src/components/forms/RowEditFooter.vue

```diff
@@ -11,58 +11,53 @@
     <div class="d-flex" style="gap: 0.5rem">
       <slot></slot>
       <ButtonAlt @click="$emit('cancel')"> Close</ButtonAlt>
-      <ButtonOutline @click="$emit('saveDraft')"> Save draft</ButtonOutline>
       <Tooltip
-        name="disabled-tooltip"
-        :value="
-          isSaveDisabled ? 'Saving is only possible on the last chapter' : ''
-        "
+        name="disabled-save-tooltip"
+        :value="saveDisabledMessage ? saveDisabledMessage : ''"
       >
-        <ButtonAction @click="$emit('save')" :disabled="isSaveDisabled">
+        <ButtonOutline
+          @click="$emit('saveDraft')"
+          :disabled="Boolean(saveDisabledMessage)"
+        >
+          Save draft
+        </ButtonOutline>
+      </Tooltip>
+      <Tooltip
+        name="disabled-save-tooltip"
+        :value="saveDisabledMessage ? saveDisabledMessage : ''"
+      >
+        <ButtonAction
+          @click="$emit('save')"
+          :disabled="Boolean(saveDisabledMessage)"
+        >
           Save {{ tableName }}
         </ButtonAction>
       </Tooltip>
     </div>
   </div>
 </template>
 
-<script>
+<script setup lang="ts">
```

Comment:
Ah you setup fanboy versus options, LOL.

Reply:
Guilty 😅